test/parallel/test-util-inspect.js

```diff
@@ -2806,35 +2807,94 @@ assert.strictEqual(
 }
 
 {
+  const originalCWD = process.cwd();
+
+  process.cwd = () => (process.platform === 'win32' ?
+    'C:\\workspace\\node-test-binary-windows%20js-suites-%percent-encoded\\node' :
+    '/home/user%20directory/repository%encoded/node');
+
   // Use a fake stack to verify the expected colored outcome.
   const stack = [
     'TypedError: Wonderful message!',
-    '    at A.<anonymous> (/test/node_modules/foo/node_modules/bar/baz.js:2:7)',
+    '    at A.<anonymous> (/t/node_modules/foo/node_modules/bar/baz.js:2:7)',
     '    at Module._compile (node:internal/modules/cjs/loader:827:30)',
     '    at Fancy (node:vm:697:32)',
     // This file is not an actual Node.js core file.
     '    at tryModuleLoad (node:internal/modules/cjs/foo:629:12)',
     '    at Function.Module._load (node:internal/modules/cjs/loader:621:3)',
     // This file is not an actual Node.js core file.
-    '    at Module.require [as weird/name] (node:internal/aaaaa/loader:735:19)',
+    '    at Module.require [as weird/name] (node:internal/aaa/loader:735:19)',
     '    at require (node:internal/modules/cjs/helpers:14:16)',
+    '    at Array.forEach (<anonymous>)',
+    `    at ${process.cwd()}/test/parallel/test-util-inspect.js:2760:12`,
+    `    at Object.<anonymous> (${process.cwd()}/node_modules/hyper_module` +
+      '/folder/file.js:2753:10)',
```

Comment:
FYI you should be able to keep it as one line, the linter doesn't enforce the same max line length for lines with template strings.
```suggestion
    `    at Object.<anonymous> (${process.cwd()}/node_modules/hyper_module/folder/file.js:2753:10)`,
```